src/services/daam/backend/deleteApplication.ts

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { ExtendedSession } from "@/app/api/auth/auth.types";
+import { decrypt } from "@/utils/encryption";
+import axios, { AxiosResponse } from "axios";
+
+export const makeDeleteApplication = (daamUrl: string) => {
+  return async (
+    applicationId: string,
+    session: ExtendedSession
+  ): Promise<AxiosResponse<void>> => {
+    return await axios.delete(
+      `${daamUrl}/api/v1/applications/${applicationId}`,
```

Comment:
important: as we discussed, this endpoint needs be changed to `POST /api/v1/applications/${applicationId}/delete`, due to current unknown bug on endpoint mapping in quarkus native-compiled code.